packages/swing-store/src/transcriptStore.js

```diff
@@ -18,7 +18,8 @@ import { createSHA256 } from './hasher.js';
  *   rolloverSpan: (vatID: string) => number,
  *   rolloverIncarnation: (vatID: string) => number,
  *   getCurrentSpanBounds: (vatID: string) => { startPos: number, endPos: number, hash: string, incarnation: number },
- *   deleteVatTranscripts: (vatID: string) => void,
+ *   stopUsingTranscript: (vatID: string) => void,
+ *   deleteVatTranscripts: (vatID: string, budget?: number) => { done: boolean, cleanups: number },
```

Comment:
Suggestion: rename "cleanups" for clarity/comprehensibility.
```suggestion
 *   deleteVatTranscripts: (vatID: string, budget?: number) => { done: boolean, count: number },
```